README.md

```diff
@@ -255,6 +255,8 @@ Each analysis definition is a map that has some properties common to all analysi
 
   If the exclude field has not been defined no extra exclusions will be applied.
 
+- **Display name**: Optional. If provided, this is the name displayed drop-down list under "Start Processing". Otherwise the name in the defined by the analysis YAML node.
```

Comment:
Suggested update to correct grammar:
"Optional. If provided, this is the name displayed in the drop-down list under "Start Processing". Otherwise, the name defined by the analysis YAML node will be used."

Reply:
Fixed